### DISCUSSION TITLE
chp anti-adblock

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uAssets#19650] [General]
for reporting chp anti-adblock-using sites without interrupting other reports

(NSFW) `https://subindojav.cc/`

**Comment by mapx-** [comment]
https://github.com/uBlockOrigin/uAssets/commit/67c464bc122ff231331ac898e0e585a07ccc5ad3

**Comment by richie3011** [comment]
@mapx- The anti adblock Re-appear & they change to new domain `https://subindojav.pw/`

**Reply by mapx-** [reply]
updated

**Comment by richie3011** [comment]
> updated

The website is dead now, currently for sale. I think you can remove it

**Comment by stephenhawk8054** [comment]
Removed: https://github.com/uBlockOrigin/uAssets/commit/2e4f0c67e782fc70358d53f63ac03b07b8b587ad
